steps/cas2-short-term-accommodation/application.ts

```diff
@@ -23,14 +23,16 @@ export async function submitApplication(page: Page, nomisId: string) {
 }
 
 async function startApplication(page: Page) {
-    await page.getByRole('link', { name: 'Start a new referral' }).click()
+    await page.getByRole('link', { name: 'Start a new application' }).click()
+    await expect(page).toHaveTitle(/Home - Short-Term Accommodation \(CAS-2\)/)
```

Comment:
You don't need regular expression matching here, so might be simpler to use a plain string which would save you having to escape the brackets?
```suggestion
    await expect(page).toHaveTitle('Home - Short-Term Accommodation (CAS-2)')
```

Reply:
Sure, done